README.md

```diff
@@ -1,63 +1,14 @@
-# ExpenseTrackerUi
+# Expense Tracker UI
 
-<a alt="Nx logo" href="https://nx.dev" target="_blank" rel="noreferrer"><img src="https://raw.githubusercontent.com/nrwl/nx/master/images/nx-logo.png" width="45"></a>
+Expense Tracker UI is the front-end for the Expense Tracker application from KTRI Industries.
+The Expense Tracker application allows for easy monitoring of expenses, while also providing insights and analytics into personal and household spending habits.
 
-✨ **This workspace has been generated by [Nx, a Smart, fast and extensible build system.](https://nx.dev)** ✨
+## The tech stack
 
+Expense Tracker UI is an Angular application created as an [Nx stand-alone project](https://nx.dev/getting-started/tutorials/angular-standalone-tutorial).
+It uses [NgRx](https://ngrx.io/) for state management, [Jest](https://jestjs.io/) for unit testing and [Cypress](https://www.cypress.io/) for end-to-end testing.
 
-## Start the app
+We opted to use Nx because:
```

Comment:
The link to the Nx standalone project page includes step-by-step directions on how to generate a standalone project.
Regarding the commands to generate libs etc, I don't think it's necessary to list them in our README since it's more related to Nx documentation, rather than specific to our project.

When we have finalized the deployement, dev environment setup etc we can add a section with related guidelines.